src/lem/pointers.rs

```diff
@@ -92,6 +92,12 @@ impl<F: LurkField> Ptr<F> {
         Self::zero(Tag::Expr(Nil))
     }
 
+    #[inline]
+    pub fn opaque(z_ptr: ZPtr<F>) -> Self {
```

Comment:
This deserves a docstring, or a reference to the docstring of `Atom`, in particular, it would be great to what properties to expect on `hash_ptr` for such an object.